pallets/detach/src/lib.rs

```diff
@@ -371,7 +365,7 @@ pub mod pallet {
 				);
 				for authority in authorities {
 					<EthereumAuthorities<T>>::mutate(|authorities| {
-						authorities.insert(authority.clone());
+						authorities.insert(*authority);
```

Comment:
Explanation to Self: `ecds::Public` implements the `Copy` trait, that's why this is possible